src/mobu/models/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Combinations of different models."""
+
+from pydantic import BaseModel, Field
+
+from .ci_manager import CiManagerSummary
+from .flock import FlockSummary
+
+
+class CombinedSummary(BaseModel):
+    """Summary of all app state."""
+
+    flocks: list[FlockSummary] = Field(
+        ..., title="Info about all running flocks"
+    )
+    ci_manager: CiManagerSummary | None = Field(
+        None, title="Info about GitHub CI workers"
+    )
```

Comment:
For applications, I'm not a big fan of non-empty `__init__.py` files except in a handful of specific cases like SQLAlchemy schemas. I'd move this into a `mobu.models.summary` module.